docs/java/video/connectivity-service.md

```diff
@@ -0,0 +1,23 @@
+---
+id: video-tutorial-about-connectivity--for-odata-with-sap-cloudsdk-for-java
+title: "Video tutorial: Generate your type-safe OData client with SAP Cloud SDK for Java"
+sidebar_label: Destinations and Cloud Connector
+description: You will learn how to generate type-safe OData client with Cloud SDK for Java and learn about consuming Workflow service via its REST API on SAP Cloud Platform
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+---
+
+Cloud Native applications are no peace of cake when you have to connect to **On Premise** systems and deliver Apps or extensions on top of that. Well, in SAP Cloud SDK for Java we think otherwise! To prove it we provide convenient abstractions over destinations to let you seamlessly connect to your **On Premise** system from you cloud App. What to know how? Watch this comprehensive video tutorial by [Max Streifeneder](https://www.youtube.com/channel/UCkzNZP9fzLxRyhnGT2ziSRw) and check our docs on [connectivity](../features/connectivity/sdk-connectivity-destination-service)
```

Comment:
```suggestion
Cloud Native applications are no piece of cake when you have to connect to **On Premise** systems and deliver Apps or extensions on top of that.
Well, in SAP Cloud SDK for Java we think otherwise!
To prove it we provide convenient abstractions over destinations to let you seamlessly connect to your **On Premise** system from your cloud app.
What to know how?
Watch this comprehensive video tutorial by [Max Streifeneder](https://www.youtube.com/channel/UCkzNZP9fzLxRyhnGT2ziSRw) and check our docs on [connectivity](../features/connectivity/sdk-connectivity-destination-service)
```